pkg/util/utils.go

```diff
@@ -71,7 +69,7 @@ func Retry(attempts int, callback func() error) (err error) {
 	return RetryAfter(attempts, callback, 0)
 }
 
-func ValidateProxyURI(uri string) bool {
+func ValidProxyURI(uri string) bool {
 	return govalidator.IsURL(uri)
```

Comment:
true. I basically kept the existing validation. That said, I checked against Go's internal checks there and they also just verify that the URL swell formed. They don't enforce a port.